File: problems/TravelingSalesman/sample/type6/cpp/main.cpp
#include <iostream>
#include <vector>
using namespace std;

class TravelingSalesman
{
public:
    auto solve (
        const int N, 
        const vector<vector<int>> c)
    {
        vector<int> v(N);
        
        for (int i = 0; i < N; i++) {
            v[i] = i;
        }

        return v;
    }
};

int main ()
{
    int N; 
    cin >> N;

    vector<vector<int>> c(N, vector<int>(N, 0));
    for (int i = 0; i < N; i++) {
        for (int j = 0; j < N; j++) {
            cin >> c[i][j];
        }
    }

    TravelingSalesman ts;
    vector<int> v = ts.solve(N, c);
    for (auto vt: v) {
        cout << vt << endl;
    }

    cout.flush();
    
    return 0;
}

File: problems/TravelingSalesman/sample/type6/python/main.py
import sys

class TravelingSalesman:
    def solve (self, N, c):
        v = []
        for i in range(N):
            v.append(i)
        return v

def main():
    N = int(input())
    c = []
    for i in range(N):
        ct = list(map(int,input().split()))
        c.append(ct)
    ts = TravelingSalesman()
    v = ts.solve(N, c)
    for i in v:
        print(i)
    sys.stdout.flush()

if __name__ == '__main__':
    main()
